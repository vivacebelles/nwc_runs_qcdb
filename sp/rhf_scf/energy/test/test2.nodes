===2018-09-27-04:16:09 PM run nodes===
compute-0-13 
