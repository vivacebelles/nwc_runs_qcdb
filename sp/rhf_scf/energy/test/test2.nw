echo

memory 400 mb
geometry noautosym nocenter units Angstrom
O     0.000000000000    0.000000000000   -0.065638538099
H     0.000000000000   -0.757480611647    0.520865616174
H    -0.000000000000    0.757480611647    0.520865616174
end

basis spherical
* library cc-pvdz
end

task scf energy
