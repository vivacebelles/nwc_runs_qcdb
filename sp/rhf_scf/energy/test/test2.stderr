cp: cannot stat `/home/alolinco/nwc_runs_qcdb/sp/rhf_scf/energy/test/*.hess': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test2/*.hess': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test2/ecce*': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test2/*.log': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test2/*.out': No such file or directory
